packages/core/build/build.tokens.ts

```diff
@@ -114,6 +115,7 @@ ${tokens.map(token => `  --cds-${camelCaseToKebab(token.name)}: ${convertCSSValu
 }`;
 
   fs.writeFileSync(path, cssTokens);
+  fs.writeFileSync(path.replace('.css', '.min.css'), csso.minify(cssTokens).css);
```

Comment:
Since the token build script already compiles/generates the CSS it simplifies rollup to have it go ahead and build the minified version